### ISSUE TITLE
Default value for loss_function

### ISSUE THREAD

**Issue by dlihhats** [JohnLangford/vowpal_wabbit#1159]
The documentation says that the default value for the loss_function parameter is 'squared'. However, the code for Python vowpal_wabbit in sklearn_vw.py sets 'logistic' as the default value.

**Comment by JohnLangford** [comment]
@gramhagen can you comment?

**Comment by gramhagen** [comment]
There are three classes in the sklearn wrapper:
2 use the vw default (squared) loss function: VW, VWRegressor
1 overrides the default loss function with logistic: VWClassifier

It was intentional to use logistic loss as the default for the classifier, since that typically makes the most sense for a classification setting and is consistent with sklearn. The other two classes are more inline with the default use case of vw. Either VWRegressor and VWClassifier can be used in conjunction with sklearn functions. The base VW estimator can be used directly with some loss of sklearn functionality, but as a thinner wrapper around the underlying vw model.

**Comment by MLWave** [comment]
Updated documentation at https://github.com/JohnLangford/vowpal_wabbit/wiki/Loss-functions to make a note of default override.

**Comment by dlihhats** [comment]
> The Python wrapper overrides the default squared loss with logistic loss when using VWClassifier.
It's probably worth noting that the Python wrapper also seems to be overriding the default squared loss with logistic loss when using VW with the `oaa` option (VWClassifier only supports binary classification currently). If @gramhagen could confirm this, it would be great. Thank you!

**Comment by JohnLangford** [comment]
Thanks all :-)
